resources/views/welcome-employee.blade.php

```diff
@@ -0,0 +1,39 @@
+@extends ('layouts.main_employee')
```

Comment:
I left it just so the page won't be completely empty, but it has been removed. Now there is only a header and breadcrumbs.